pkg/client/broker.go

```diff
@@ -0,0 +1,68 @@
+// (C) Copyright 2024 Hewlett Packard Enterprise Development LP
+
+package client
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"log"
+
+	consts "github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/common"
+	"github.com/HewlettPackard/hpegl-vmaas-cmp-go-sdk/pkg/models"
+)
+
+// BrokerAPIService is a service that provides methods to interact with the broker API
+type BrokerAPIService struct {
+	Client APIClientHandler
+	Cfg    Configuration
+}
+
+// GetMorpheusDetails returns Morpheus details to terraform
+func (a *BrokerAPIService) GetMorpheusDetails(ctx context.Context) (models.MorpheusDetails, error) {
+	// Get the service instance ID and Morpheus URL
+	ServiceSubscriptionDetailsResp := models.SubscriptionDetailsResponse{}
+	serviceSubscriptionDetailsAPI := &api{
+		method:                 "GET",
```

Comment:
could `http.MethodGET` constant be used here?

Reply:
It could indeed Gandharva.  I'll make that change.  Thanks for the review.